packages/rust-analyzer/build.sh

```diff
@@ -0,0 +1,18 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/rust-analyzer/rust-analyzer
+TERMUX_PKG_DESCRIPTION="A Rust compiler front-end for IDEs"
+TERMUX_PKG_LICENSE="MIT, Apache-2.0"
+TERMUX_PKG_LICENSE_FILE="LICENSE-MIT, LICENSE-APACHE"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=2021-06-07
```

Comment:
Please change so that the version is 20210607, and use something like `${TERMUX_PKG_VERSION:0:4}-${TERMUX_PKG_VERSION:4:2}-${TERMUX_PKG_VERSION:6:2}` in the source url. [Hyphens do not work in version strings](https://www.debian.org/doc/debian-policy/ch-binary.html#the-version-of-a-package).

Reply:
Done .